.gitignore

```
@@ -0,0 +1,2 @@
+__pycache__
+*.egg-info
```

```suggestion
*.egg-info
/build

```

1. Please ignore the `/build` directory as well.
2. Please add a `newline` at the end of the file